exercises/pangram/pangram_test.rb

```diff
@@ -19,6 +19,16 @@ def test_pangram_with_only_lower_case
     assert Pangram.is_pangram?(str)
   end
 
+  def test_pangram_with_underscore_character
+    str = 'the quick brown_fox jumps over the lazy dog'
+    assert Pangram.is_pangram?(str)
+  end
+
+  def test_pangram_with_sentence_containing_numbers
+    str = 'the qu1ck br0wn fox jumps over the lazy dog'
```

Comment:
The purpose of this test is to replace the letters with a corresponding number and make sure the solution does not think the string is a pangram. With this in mind, I think its ok. Especially if the method name were updated to something like `test_letters_replaced_with_numbers`.

At worst its an exercise in paying attention to detail, which is a good skill to have as a programmer.